bin/describe/describe_pp.ml

```diff
@@ -18,15 +18,22 @@ let pp_with_ocamlc sctx project pp_file =
   let+ () =
     Process.run
       ~display:!Clflags.display
+      ~dir:(Path.build (Super_context.context sctx).build_dir)
       ~env:(Super_context.context_env sctx)
       Strict
       (Super_context.context sctx).ocaml.ocamlc
-      [ "-stop-after"; "parsing"; "-dsource"; Path.to_string pp_file; "-dump-into-file" ]
+      [ "-stop-after"
+      ; "parsing"
+      ; "-dsource"
+      ; Path.to_string (Path.drop_optional_build_context_maybe_sandboxed pp_file)
+      ; "-dump-into-file"
+      ]
   in
   match Path.stat dump_file with
   | Ok { st_kind = S_REG; _ } ->
     Io.cat dump_file;
-    Path.unlink_no_err dump_file
+    Path.unlink_no_err dump_file;
+    Path.rm_rf dump_file
```

Comment:
What is the purpose of this double remove? If `unlink_no_err` succeeds `rm_rf` will catch `ENOENT` and return unit, it can never be a directory anyway and otherwise it will call `unlink` (without catching exceptions), so if exposing the exception is the desired effect, `Path.unlink` can be used directly instead of `Path.unlink_no_err`.

Reply:
Yes, good point. Fixed.